federation/cluster/common.sh

```diff
@@ -87,7 +87,18 @@ function create-federation-api-objects {
       fi
     fi
 
-    export FEDERATION_SERVICE_CIDR=${FEDERATION_SERVICE_CIDR:-"10.10.0.0/24"}
+    if [[ "${KUBERNETES_PROVIDER:-}" == "gce" || "${KUBERNETES_PROVIDER:-}" == "gke" ]]; then
+      PD_PLUGIN_NAME='kubernetes.io/gce-pid'
```

Comment:
I assume you wanted "pd" not "pid".  Your UNIX is showing.  But you don't need to say anything special here.  Any value will do, for now - that's what makes it Alpha.  It only works on AWS, GCE, and OpenStack, and it knows what kind of volume to provision based on that.